### ISSUE TITLE
CR: Platform - Remove Aria-Label and Add Alt Text to Logo in Header

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/cgov-digital-platform#4500]
### Request Description
The logo index link in the header should be updated to be more accessible than it is currently.  Working with our PS accessibility resources, the image of the logo should have alt text instead of using aria-label.

[W3C Source for requirement](https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo)
[Webaim Source for requirement](https://webaim.org/techniques/alttext/#logos)

Across the digital platform, the logo link image needs to have `alt-text` that is descriptive, such as "National Cancer Institute Homepage".

### Expected Functionality
- The logo link image on the header shall have the `alt-text` added to the `<img src` with text that is similar and appropriate for each logo. An example on the cancer.gov homepage is "National Cancer Institute Home Page"

### Current Functionality
- The `aria-label` is on the `<a` section of the logo and that is not the best practice for accessibility.

**Comment by alinai** [comment]
@andyvanavery31 , should Logo in Header for all content types should show the same alt-text ( alt = National Cancer Institute Home Page)? 

![image](https://github.com/user-attachments/assets/95818a2b-8e93-4f78-b654-208cdde3a151)

**Comment by andyvanavery31** [comment]
@alinai this behavior is correct because wherever we have the NIH National Cancer Institute Logo in the header, the link in there returns the user to the National Cancer Institute Home Page. However, we will need to confirm that this update correctly works for DOC sites too because they will have their own homepage names (e.g., DCEG would need to move their "National Cancer Institute Division of Cancer Epidemiology and Genetics Home Page" on their aria-label to the alt-text, and this should be done automatically across the entire platform)

**Comment by alinai** [comment]
> @alinai this behavior is correct because wherever we have the NIH National Cancer Institute Logo in the header, the link in there returns the user to the National Cancer Institute Home Page. However, we will need to confirm that this update correctly works for DOC sites too because they will have their own homepage names (e.g., DCEG would need to move their "National Cancer Institute Division of Cancer Epidemiology and Genetics Home Page" on their aria-label to the alt-text, and this should be done automatically across the entire platform)

@blairlearn , is there any way for us to verify DCEG and other doc sites on ODE?

**Comment by blairlearn** [comment]
> @blairlearn , is there any way for us to verify DCEG and other doc sites on ODE?

That's not quite the right question.  The doc sites run on the same code as www.  So from that viewpoint, cgov is also a microsite.

What this means is, you want to verify that the _functionality_ needed to support a given DOC's requirements works correctly on the ODE.

**Comment by dlescarbeau** [comment]
This is a change request that alters the implementation of this ticket's requirements: https://github.com/NCIOCPL/cgov-digital-platform/issues/3268
